pkg/v4/resource/certconfig/create.go

```diff
@@ -23,7 +27,7 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 		r.logger.LogCtx(ctx, "level", "debug", "message", "creating certconfigs")
 
 		for _, certConfigToCreate := range certConfigsToCreate {
-			_, err = r.g8sClient.CoreV1alpha1().CertConfigs(v1.NamespaceDefault).Create(certConfigToCreate)
+			_, err = r.g8sClient.CoreV1alpha1().CertConfigs(objectMeta.Namespace).Create(certConfigToCreate)
```

Comment:
Why not just using the namespace of `certConfigToCreate`?

Reply:
Makes sense. I'll change.